packages/blade/src/tokens/global/typography.ts

```diff
@@ -4,6 +4,7 @@ import type { FontFamily } from './fontFamily/types';
 type FontWeight = {
   regular: 400;
   bold: 700;
+  semibold: 600;
```

Comment:
is this the correct weight for semibold?

Reply:
Should be, although because we didn't found the semibold font on google fonts, not sure how should we proceed now.

Reply:
Even on https://fontsource.org/fonts/lato there's only 400,700,900. 

So in this case we might have to treat the black (900) as "bold" and 700 as semibold. 

@saurav12